src/mscorlib/src/System/IO/File.cs

```diff
@@ -36,6 +36,11 @@ namespace System.IO {
     [ComVisible(true)]
     public static class File
     {
+        internal const int GENERIC_READ = unchecked((int)0x80000000);
+        private const int GENERIC_WRITE = unchecked((int)0x40000000);
+        private const int FILE_SHARE_WRITE = 0x00000002;
+        private const int FILE_SHARE_DELETE = 0x00000004;
+
```

Comment:
Is the `class File` actually used anywhere in CoreLib? Would it be better to just exclude it from build?